docs/user_guide/source/faq/faq.rst

```diff
@@ -35,11 +35,48 @@ FAQs Answered
 Can I use the same library for MPI and non-MPI code?
 ----------------------------------------------------
 
-Short answer: No.
+Short answer: Yes. 
```

Comment:
@pnorbert something like "since version 2.6.0" would clarify expectations for older versions. Thanks!